Demo/Demo/Features/FeatureBaseViewController.swift

```diff
@@ -0,0 +1,91 @@
+import UIKit
+
+class FeatureBaseViewController: UIViewController {
+
+    let containerView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    let titleLabel: UILabel = {
+        let label = UILabel()
+        label.textAlignment = .center
+        label.numberOfLines = 0
+        label.lineBreakMode = .byWordWrapping
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureBottomView()
+    }
+
+    func configureBottomView() {
+        view.addSubview(containerView)
+        containerView.addSubview(titleLabel)
+
+        NSLayoutConstraint.activate([
+            containerView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
```

Comment:
Not sure why we need this containerView?

Reply:
Typically, I've seen using a container for these types of labels to make adding things to them in the future easier as well as alignment since the text of the label will need to be dynamic. That said, we can certainly remove it and if we get to that point where it makes sense, re-add it.